src/utils/filterUtils.ts

```diff
@@ -0,0 +1,25 @@
+import { Restaurant } from "../data/restaurant.js";
+
+export const filterRestaurants = (
+  restaurants: Restaurant[],
+  category: string
+) => {
+  if (category === "") return restaurants;
+  return restaurants.filter((r) => r.categoryTitle === category);
+};
+
+export const sortRestaurants = (
+  restaurants: Restaurant[],
+  sortBy: string
```

Comment:
name과 distance만 받을거라면 타입을 좁혀두면 좋을거 같아요

```suggestion
  sortBy: "name" | "distance"
```

Reply:
https://github.com/woowacourse/javascript-lunch/pull/241/commits/cd2200dc7c6b68ef9c97043de6c8c6068319d48f 필터링 및 정렬 선택 가능 타입을 좁혔습니다!

- 필터링 가능 타입: 미선택, 한식, 중식, 일식, 양식, 아시안, 기타
- 정렬 가능 타입: name, distance